pkg/operator/ceph/csi/spec.go

```diff
@@ -361,23 +361,22 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 	tp.Param.MountCustomCephConf = CustomCSICephConfigExists
 
 	if EnableRBD {
+		tp.CsiComponentName = nodePlugin
```

Comment:
we should check if csi log rotation is enabled then only append these values? 		

if tp.CSILogRotation {

Reply:
IMO we dont need to it, those values will be rendered only if the `tp.CSILogRotation` is set.

Reply:
make sense